packages/bun-types/package.json

```diff
@@ -1,25 +1,38 @@
 {
   "name": "bun-types",
-  "repository": "https://github.com/oven-sh/bun",
   "license": "MIT",
+  "main": "",
```

Comment:
is main being "" intentional?

Reply:
Yes, this is to show an error to people trying to import the package in js instead of just using it for the types